src/Widgets/AppContainers/ListPackageRowGrid.vala

```diff
@@ -28,8 +28,13 @@ public class AppCenter.Widgets.ListPackageRowGrid : AbstractPackageRowGrid {
     construct {
         package_summary = new Gtk.Label (null) {
             ellipsize = Pango.EllipsizeMode.END,
+            hexpand = true,
+            lines = 3,
```

Comment:
Can we do 2 lines here instead of 3? that seems like just a ton of wrapping. We end up in some cases with only 1 or 2 words per line